packages/hop-node/src/watchers/classes/Bridge.ts

```diff
@@ -829,6 +829,7 @@ export default class Bridge extends ContractBase {
   }
 
   getConfigBonderAddress (): string {
-    return globalConfig?.bonders?.[this.tokenSymbol]?.[0]
+    const sourceBonders = globalConfig?.bonders?.[this.tokenSymbol]?.[this.chainSlug]
+    return sourceBonders?.[Object.keys(sourceBonders)[0]]
```

Comment:
This is used to estimate gas for transactions, so we need to ensure that the correct bonder is used for the estimations. So I believe this needs to return:

`return sourceBonders?.[this.destination]` or something like that.

With dedicated routes, we cannot default to `[0]` when

Reply:
Resolved by https://github.com/hop-protocol/hop/commit/98c910a15e4b6e3537df362b44283f62f2164323